src/components/modal/Modal.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Modal as RNModal, View, Text, TouchableOpacity, ViewProps } from 'react-native';
+
+import { styles } from './modal.styles';
+
+interface ModalProps extends ViewProps {
+  isVisible: boolean;
```

Comment:
order alphabetically

Reply:
Can we take inspiration from https://baseweb.design/components/modal/ or other populate open source library on how to create abstractions for modal component please?